src/scss/components/intro.scss

```diff
@@ -0,0 +1,57 @@
+.Intro {
+  background-image: url('../static/assets/FormaCirculoAzul.png');
```

Comment:
all the images files have the convention like this:
1. always in lower case. 
2.pull apart the words with (-) hyphen.
3. no more than 2 words